EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/basket/basket_features/basket_storage_time.md

# Basket storage time

> The time basket is stored depends on whether a basket belongs to an anonymous user or a user logged in.

Reviewer: I would go along the lines of:
"The time for which a basket is stored depends on whether the basket..."

> The time basket is stored depends on whether a basket belongs to an anonymous user or a user logged in.

Reviewer 2: ```suggestion
The time for which a basket is stored depends on whether a basket belongs to an anonymous user or a logged user.
```

> A basket for a logged-in customer is stored for ever.

Reviewer: forever

> A basket for a logged-in customer is stored for ever.

Reviewer 2: ```suggestion
A basket for a logged user is stored forever.
```

A basket for an anonymous user is stored for 120 hours by default.
You can configure a different value:

``` yaml
parameters: 
    ses_basket.default.validHours: 120
```

> You can use the `silversolutions:baskets:clear` command to clean up anonymous baskets which have been expired:

Reviewer: ...to delete anonymous baskets that have expired:

> You can use the `silversolutions:baskets:clear` command to clean up anonymous baskets which have been expired:

Reviewer 2: ```suggestion
You can use the `silversolutions:baskets:clear` command to remove expired anonymous baskets:
```
